docs/upgrade.html

```diff
@@ -136,9 +136,11 @@ <h5><a id="upgrade_420_notable" href="#upgrade_420_notable">Notable changes in 4
                 settings.
             </li>
             <li>
-                The <code>cleanup.policy</code> is empty and <code>remote.storage.enable</code> is set to true, the
+                If <code>cleanup.policy</code> is empty and <code>remote.storage.enable</code> is set to true, the
                 local log segments will be cleaned based on the values of <code>log.local.retention.bytes</code> and
                 <code>log.local.retention.ms</code>.
+                <br>
+                Note that <code>cleanup.policy</code> supports empty values, which means infinite retention.
```

Comment:
We could remind users that this use case is equivalent to setting `retention.ms=-1` and `retention.bytes=-1`